lib/upload.js

```diff
@@ -1014,6 +1097,96 @@ function splitSizeIntoParts (totalSize, partCount) {
   return parts
 }
 
+/**
+ * Build the chunk staggers.
+ *
+ * NOTE: Must be kept in sync with `splitSizeIntoParts` in skynet-js.
+ */
+function buildChunkStaggers (totalSize, partCount, chunkSize) {
+  // The lists of chunks for each part.
+  const partsChunks = []
+  for (let i = 0; i < partCount; i++) {
+    partsChunks.push([])
+  }
+  // The leftover size that must go into the last part.
```

Comment:
`nit` comment says it's the size but it's not, probably updated w/o updating comment